test/blackbox-tests/test-cases/cinaps/multiple-cinaps.t

```diff
@@ -0,0 +1,26 @@
+Multiple cinaps stanzas in the same dune file
+
+  $ cat > dune-project <<EOF
+  > (lang dune 2.8)
+  > (using cinaps 1.0)
+  > EOF
+
+  $ cat > dune <<EOF
+  > (cinaps (files *.ml))
+  > (cinaps (files *.mli))
+  > EOF
+
+  $ touch x.ml x.mli
+
+  $ dune runtest --diff-command diff 2>&1 | sed -E 's/[^ ]+sh/\$sh/'
```

Comment:
I found this recently:

```
export BUILD_PATH_PREFIX_MAP="sh=$(which sh):$BUILD_PATH_PREFIX_MAP"
```

which is a nice way to handle this rewriting. Maybe we should allow to specify a "setup.sh" script in the cram stanza to setup the environment in all tests.

Reply:
> I kind of prefer letting the user select their own "runner" for tests. This would also make it easier to use more portable shells in some situations.

Allowing to select an alternative runner seems fine, but I would still prefer being able to specify a setup script. Specifying a setup script by changing the runner might be a bit awkward, and a setup script is also a simpler feature to support. If we truly want to allow users to select an alternative shell, we might have to rethink the way cram tests work given that we generate an intermediate shell script in sh syntax.